confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -164,12 +168,14 @@ public ConfigManager() throws IOException {
     this.udfManager = new UDFManager(this, udfInfo);
     this.loadManager = new LoadManager(this);
     this.consensusManager = new ConsensusManager(this, stateMachine);
+    this.dataNodeRemoveManager = new DataNodeRemoveManager(this);
```

Comment:
Does dataNodeRemoveManager has any data persisted in disk? If it has, it should be initialized after ConsensusManager.